[DOCS] Reference to Hugging Face (#24961)
openvinotoolkit/openvino: docs/articles_en/documentation/legacy-features/transition-legacy-conversion-api/legacy-conversion-api/[legacy]-supported-model-formats/[legacy]-conversion-tutorials/convert-tensorflow-efficient-det.rst

    The color channel order (RGB or BGR) of an input data should match the channel order of the model training dataset. If they are different, perform the ``RGB<->BGR`` conversion specifying the command-line parameter: ``--reverse_input_channels``. Otherwise, inference results may be incorrect. For more information about the parameter, refer to the **When to Reverse Input Channels** section of the :doc:`Converting a Model to Intermediate Representation (IR) <../../[legacy]-setting-input-shapes>` guide.

OpenVINO toolkit provides samples that can be used to infer EfficientDet model.
For more information, refer to the :doc:`Open Model Zoo Demos <../../../../../../omz_demos>`.

.. important::

>    Due to the deprecation of Open Model Zoo, models in OpenVINO IR format have been gradually

Reviewer: ```suggestion
   Due to the deprecation of Open Model Zoo, models in the OpenVINO IR format are now
```

Author: Applied with https://github.com/openvinotoolkit/openvino/pull/24961/commits/4f93fbdcd645969a2b8b73f62d70888431ec6a9d
